app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,11 @@
+class RestaurantsController < ApplicationController
+
+  def index
+    @restaurants=  Restaurant.all
```

Comment:
Listen to the Hound! Future PR managers may not be as relaxed as I am about not conforming to the Ruby styleguide.